hs/WebParsing/ParsingHelp.hs

```diff
@@ -1,8 +1,8 @@
 {-# LANGUAGE OverloadedStrings #-}
 module WebParsing.ParsingHelp
    (CoursePart,
-    (-:),
     (~:),
+    (-:),
```

Comment:
I would tend to agree. The few locations where this is used can probably be re-written.